src/components/NewMessageForm/NewMessageForm.vue

```diff
@@ -921,15 +921,16 @@ export default {
 			position: relative;
 		}
 
+		// Override NcRichContenteditable styles
 		&__richContenteditable {
```

Comment:
```suggestion
 	 & 	&__richContenteditable {
```
I'm not a fan of using `!important`, while it could cause some troubles in future with style overriding.
Probably mentioned above nesting should be enough

Reply:
Agree. There is a better solution.